pages/collection/[id].vue

```diff
@@ -284,6 +284,12 @@
             :description="project.description"
             :downloads="project.downloads ? project.downloads.toString() : '0'"
             :follows="project.followers ? project.followers.toString() : '0'"
+            :featured-image="
```

Comment:
It should just filter by featured and show nothing if none are featured, it's a backend bug that the first non-featured one is shown in search.

Reply:
So I just need to use find() function?

Reply:
yeah, to get the first gallery image that is featured. if none, then show none

Reply:
Done (I think) ^^.

Reply:
Now that's fixed ^^.